openwisp_utils/admin_theme/static/admin/css/openwisp.css

```diff
@@ -658,6 +658,10 @@ input[type='button']:hover,
   float: none;
   margin-bottom: 26px;
 }
+.no-auth .title-wrapper h1{
+  float: none;
+  margin: 0;
+}
```

Comment:
I will be removing these lines. It has been fixed here. https://github.com/openwisp/openwisp-utils/pull/226